src/storage/src/row_serde/row_serde_util.rs

```diff
@@ -111,3 +113,49 @@ pub fn parse_raw_key_to_vnode_and_key(raw_key: &[u8]) -> (VirtualNode, &[u8]) {
     let vnode = VirtualNode::from_be_bytes(vnode_bytes.try_into().unwrap());
     (vnode, key_bytes)
 }
+
+/// used for streaming table serialize and deserialize
+/// todo(wcy-fdu): remove `RowSerde` trait after all executors using streaming table.
+pub fn serialize(vnode: VirtualNode, pk: &[u8], row: Row) -> Result<(Vec<u8>, Vec<u8>)> {
+    let mut value_bytes = vec![];
+    let key = [vnode.to_be_bytes().as_slice(), pk].concat();
+    for cell in &row.0 {
+        value_bytes.extend(serialize_datum(cell)?);
+    }
+    let res = (key, value_bytes);
+    Ok(res)
+}
```

Comment:
after read the current code in state table, I think maybe we can just change it to just serialize the value. the pk part we can use the `StateTable::serialize_pk_with_vnode`. Also after that we can remove `compute_vnode_by_row`